### DISCUSSION TITLE
backup linux system at rest using rsync + reverse mode

### DISCUSSION THREAD

**Discussion by lsching17** [rfjakob/gocryptfs#714] [General]
I would like to use rsync + reverse mode to backup a linux system at rest  to the cloud. The gocryptfs version is 2.2.1

Is the following supported in reverse mode? are they encrypted? Do i need to add special mount option?

- ACLs
- xattrs
- special files e.g. inside /dev
- symlink/hardlink

I want to know if i can recover the linux system by rsync + decrypt when disaster recovery

**Comment by rfjakob** [comment]
Sorry for the delay. Disaster recovery like this may or may not work because of the limitations below. If you do try it, I think you want to use `--one-file-system` .


> ACLs

No

> xattrs

No

> special files e.g. inside /dev

Yes

> symlink/hardlink

Yes and yes
